x/projects/keeper/creation.go

```diff
@@ -85,8 +85,13 @@ func (k Keeper) CreateProject(ctx sdk.Context, subAddr string, projectData types
 	}
 
 	// project name per subscription is unique: check for duplicates
+	// we also check !isDeleted because when a new subscription is created
+	// in the same epoch that a subscription from the same creator was expired,
+	// we'll find the old admin project but since it's deleted we want to ignore
+	// it and allow creating a new admin project
 	var emptyProject types.Project
-	if found := k.projectsFS.FindEntry(ctx, project.Index, epoch, &emptyProject); found {
+	_, isDeleted, _, found := k.projectsFS.FindEntryDetailed(ctx, project.Index, epoch, &emptyProject)
+	if found && !isDeleted {
```

Comment:
yes there will be two entries with the same index. The differences between them:
Entry A: `Refcount`=0, `DeleteAt`=blockWithinTheEpoch, `StaleAt`= blockWithinTheEpoch+stalePeriod, `IsLatest`=false
Entry B: `Refcount`=1, `DeleteAt`=MaxUint64, `StaleAt`= MaxUint64, `IsLatest`=true

since entry B is the latest, it will be found first in every query. All the unit tests pass but I do admit that it may require extra care. @Yaroms what do you think?

Reply:
After talks with @Yaroms the whole purpose of this PR changed. Now all deletions are triggered from the next epoch, so this code doesn't exist anymore.